work_plan.md

```diff
@@ -17,8 +17,7 @@ Our proposal has two major components:
 * Broadening social foundations by building a platform and content for
   teaching neuroimaging with Nibabel.
 
-Code foundations
-================
+## Code foundations
```

Comment:
Don't forget - this goes into the document as plain text, not markdown.  That's why I used the underlines - they look more obvious for subheadings without being rendered.

Reply:
Gotcha will revert back